azure-functions/acbs-function/constants/deal.js

```diff
@@ -12,7 +12,17 @@ const SME_TYPE = {
   NOT_KNOWN: 'Not known',
 };
 
+const EXPIRATION_DATE = {
+  NONE: '99/99/99',
+};
+
+const PARTY = {
+  GUARANTOR: '00000141',
```

Comment:
Just curious, it's required to send this as a hardcoded value?

Reply:
Yes I'm afraid so mate